src/50_Samples_%26_Templates/Comment_Section.html

```diff
@@ -0,0 +1,135 @@
+    }
+    form.amp-form-submit-success .data-input,
+    form.amp-form-submit-success .img-data-input, 
+    form.amp-form-submit-success .button{
+        display: none;
+    }
+    form.amp-form-submit-error [submit-error] {
+        color: red;
+    }
+    .date {
+      color: #a8a3ae;
+      font-size: 12px;
+    }
+    .comment {
+        background: #fff;
+        margin: 0 0 16px 0;
+        width:100%;
+    }
+    .nomargin {
+        margin: 0;
+    }
+    .space-left {
+      margin-left: 16px;
+    }
+    .nopadding {
+      padding: 0;
+    }
+    .comment-user {
+      display: flex;
+    }
+    </style>
+
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script id="amp-access" type="application/json">
+    {
+        "authorization": [[.AuthorizationEndpoint]],
+        "pingback": "/samples_templates/comment_section/pingback?rid=READER_ID&ref=DOCUMENT_REFERRER&url=CANONICAL_URL",
+        "login": [[.LoginEndpoint]],
+        "authorizationFallbackResponse": {
+            "error": true,
+            "access": false
+        }
+    }
+    </script>
+    </head>
+    <body>
+    <!-- #### Add a Comment Section -->
+  <!--
+    Use the `amp-access` to realize a login flow. Learn more about amp-access [here](/components/amp-access). 
+    Use the `amp-form` component to realize a comment section. Learn more about amp-fomr [here](/components/amp-form).
+  -->
+  <div>
+    <span amp-access="user = 'test-user'" amp-access-hide>
+          <a href="<%host%>/samples_templates/[[.Path]]/logout" class="button button-primary space-left">Logout</a>
+    </span>            
+    <span amp-access="NOT user = 'test-user'" role="button" tabindex="0">
+          <h5>Please login to comment</h5>
+          <a on="tap:amp-access.login" class="button button-primary space-left">Login</a>  
+    </span>
+    <span amp-access="user = 'test-user'" amp-access-hide>    
+      <form method="post" action="<%host%>/samples_templates/comment_section/submit-comment" action-xhr="<%host%>/samples_templates/comment_section/submit-comment-xhr"  target="_top" >
+      <div class="full-width">
+        <h5 class="nopadding">Comments</h5>
+        <div>
+          [[range .Comments]]
```

Comment:
Why do you need comment timestamps? Can't you just manually define them in the template?